data/human/human missions.txt

```diff
@@ -7453,3 +7453,1280 @@ mission "Adelita 5"
+			`	"Good expedition." Kormak says beneath the emerging stars.`
+			`	Tavish nods, "Certainly more than any of us bargained for, but tomorrow the real work begins and who knows what wonders we will discovered in that ancient crashed ship."`
+			`	"Yes." Kormak says between bites, "Where I from songs would be sung of our deeds."`
+			choice
+				`	"Kormak, where are you from?"`
+				`	(Say nothing.)`
+					goto notathing
+
+			`	Kormak stops chewing and looks hard at you for a long moment before finally says, "It is forbidden."`
+			`	"Ah, about that," Tavish jumps in, "the thing is the Deep is more complex than most people realize. It is a society with three distinct layers."`
+			`	Tavish mimes peeling a fruit, "The top layer is the one everyone knows about. It's clean, fair, democratic, and technologically advanced. It's the Deep you hear about in holo-dramas, and see talked about on the news. Most of the Deep is a part of this layer.`
+			`	"And then," Tavish continues, "there's the middle layer. It is smaller and made up of people who practice what they call the old ways. Those who practice stone-funerals, or who pay some respect to the scattered God-Stones. It's people who still see themselves as the descendants of those first brave pilgrims that colonized the Deep. Like Hroar's Haulers."`
+			`	Tavish pinches his fingers together like he's holding his invisible fruit's core, "Finally, there's the lowest layer. Few are aware of it, and I think it may be an embarrassment to leaders of the Deep. It's made up of people like Kormak, or the Ice-Witches of Nifel." `
+			`	"The Ui Skathi." Kormak says, "Granddaughters of Skathi."`
+			`	"Was that what they called themselves?" Tavish shrugs, "Kormak's always a little sensitive about this. Anyways, they don't really make sense. How did they get to the Deep? Where did they come from? There's two prevailing theories. First, that there was a lost generational fleet that landed in the Deep earlier than anyone realized, that something went wrong, and that they degenerated in primitives."`
+			`	Kormak frowns, "Lies.`
+			`	"Or, well this is more of folklore than a theory, that thousands of years ago humans were brought to the Deep by... elves."`
+			`	"Not elves," Kormak interrupts with a hint of annoyance,"the Dream Anuabhair. That how they called in my clan's sagas. The exceedingly proud folk. Never to trust."`
+			`	Kormak sighs and turns back to you, "If ever you stand before gleaming high-timbered hall built in shadow of dark Sidhe-Stone with carved face of Skoll-Wolf. Know that is my home."`
+			`	"Enough about past. Only present matters." He says and then goes back to eating.`
+			`	Tavish laughs, "Of course. Let us discourse about less weighty matters."`
+
+			label notathing
+			`	You talk a little of unimportant and unremarkable things beneath the glittering twilight stars in the midst of an untamed alien jungle.`
+			`	And then you head to bed.`
+			choice
```

Comment:
I'm not sure whether I like this choice or not. You definitely need some way to divide between days, so I think it works.